src/xapi.c

```diff
@@ -632,19 +632,29 @@ static void throttle(void)
  */
 int xapi_set(void)
 {
-    char buffer[MSG_SIZE];
+    char* buffer;
```

Comment:
Coding style: `char *buffer` instead. :wink: (There are other declaration in the same case.)